### ISSUE TITLE
trailing comma suppoert (rust 1.22) 

### ISSUE THREAD

**Issue by colin-kiegel** [colin-kiegel/rust-pretty-assertions#16]
rust 1.22 supports trailing commas like
```rust
assert_eq!(1, 1,);
assert_ne!(1, 2,);
debug_assert_eq!(1, 1,);
debug_assert_ne!(1, 2,);
```

https://github.com/rust-lang/rust/pull/45887

**Comment by fflorent** [comment]
Yup. Probably a good first bug!

Florent

**Comment by colin-kiegel** [comment]
published with `v0.4.1`
